autoload/db.vim

```diff
@@ -620,12 +614,19 @@ endfunction
 
 function! s:vim_job.call_on_finish_if_closed() abort
   if self.close && self.exit
-    return self.on_finish(split(self.output, "\n", 1), self.exit_status)
+    let output = split(self.output, "\n", 1)
+    if len(output) && empty(output[-1])
+      call remove(output, -1)
+    endif
```

Comment:
This isn't necessary, and causes the resulting files to be `'noendofline'`.

Reply:
Do you know if same thing applies to Neovim? I just copy/pasted what you did with the system previously.

Reply:
Removed.